### PR TITLE
[Feature] Several UI Changes for aleo.tools Develop Tab

### LINKED ISSUE #747: [Feature] Several UI Changes for aleo.tools Develop Tab
## 🚀 Feature

Several changes to improve the user interface and developer experience of aleo.tools:

1. Rename the "Demo" button to "Insert Demo Program" or something similar to make its function more clear.
2. Add an info tooltip that appears on hover to specify that the program field should contain Aleo Instructions code.
3. Add an info tooltip that clarifies the "Fee Record" must be in plaintext format and provide an example record plaintext. Mention that this plaintext can be obtained by decrypting an unspent record.

## Motivation

These changes aim to make aleo.tools more dev-friendly and intuitive, particularly for newcomers who may not be familiar with the terminology or functionality.

## Implementation

1. **Demo Button Renaming**: Update the Demo button text from "Demo" to "Insert Demo Program". This is likely a small change but will have a significant impact on clarity.

2. **Info Tooltip for Program Field**: A tooltip can be added to the program field to display additional information on hover. This tooltip should specify that Aleo Instructions code is expected in this field. For example:
```diff
  <Form.Item
      label="Program"
      name="program"
+     tooltip={"This must be an Aleo Instructions program."}
      rules={[
          {
              required: true,
              message: "Please input or load an Aleo program",
          },
      ]}
  >
      <CodeEditor onChange={onProgramChange} />
  </Form.Item>
```

4. **Info Tooltip for Fee Record**: Similar to the program field, an info tooltip can be added to the "Fee Record" field. This tooltip should provide an example and mention that the fee record must be in plaintext format, and can be found in the latest transition outputs and decrypted in the Record tab. For example:
```diff
<Form.Item
    label="Fee Record"
    colon={false}
+  tooltip={`This is the record you will use to pay your tx fee and must be in plaintext format. Record plaintext can be obtained by decrypting an unspent record.
+        Ie: {
+        owner: aleo1j7...,
+        microcredits: 15000...,
+        _nonce: 30774...
+        }`}
    validateStatus={status}
>

```

**Components Affected**: Primarily the UI components responsible for the "Demo" button, program field, and "Fee Record" field.

**Are you willing to open a pull request?**
Maybe

### PR DESCRIPTION
**Pull request by SamantaTarun**
##Feature

Several changes to improve the user interface and developer experience of aleo.tools:

    Rename the "Demo" button to "Insert Demo Program" or something similar to make its function more clear.
    Add an info tooltip that appears on hover to specify that the program field should contain Aleo Instructions code.
    Add an info tooltip that clarifies the "Fee Record" must be in plaintext format and provide an example record plaintext. Mention that this plaintext can be obtained by decrypting an unspent record.

## Motivation

These changes aim to make aleo.tools more dev-friendly and intuitive, particularly for newcomers who may not be familiar with the terminology or functionality.

## Test Plan

Manual test

(Write your test plan here)

## Related PRs

Closes #747 

(Link your related PRs here)

### CODE REVIEW

**Comment by onetrickwolf** [comment]
@tarunsamanta2k20 thanks for the contribution! 

@moAlobaidi is this what you had in mind?

File: website/src/tabs/develop/Deploy.jsx [hunk 1]
```diff
@@ -282,6 +283,12 @@ export const Deploy = () => {
                 <Form.Item
                     label="Fee Record"
                     colon={false}
+                    tooltip={`This is the record you will use to pay your tx fee and must be in plaintext format. Record plaintext can be obtained by decrypting an unspent record.
+                          Ie: {
+                          owner: aleo1j7...,
+                          microcredits: 15000...,
+                         _nonce: 30774...
+                        }`}
```
Comment (moAlobaidi) [inline website/src/tabs/develop/Deploy.jsx]:
```suggestion
                    tooltip={`Use this plaintext record to pay your tx fee,
                     e.g., { owner: aleo1j7..., microcredits: 15000..., _nonce: 30774... }.
                      Obtain it by decrypting an unspent record in the 'Record' tab.`}
```

Comment (moAlobaidi) [reply website/src/tabs/develop/Deploy.jsx]: Please edit your copy to this for conciseness / clarity.

**Comment by moAlobaidi** [comment]
Welcome @tarunsamanta2k20 and thank you for putting this up, huge help! I left a comment about changing the tooltip copy, will approve once those changes are made. Cheers and welcome to the Aleo dev community 🥂

**Comment by SamantaTarun** [comment]
> Welcome @tarunsamanta2k20 and thank you for putting this up, huge help! I left a comment about changing the tooltip copy, will approve once those changes are made. Cheers and welcome to the Aleo dev community 🥂

May i know what is your comment?

**Review by moAlobaidi — APPROVED** [review]
LGTM!

**Comment by moAlobaidi** [comment]
@tarunsamanta2k20 Congratulations on your first merged PR  🎉

**Pull request merged**